tools/go-cli/go-whisk/whisk/shared.go

```diff
@@ -38,4 +38,5 @@ type Parameters *json.RawMessage
 type Limits struct {
     Timeout int `json:"timeout,omitempty"`
     Memory  int `json:"memory,omitempty"`
+    Logsize int `json:"logs"`
```

Comment:
IMHO, the JSON label "logsize" would better fit the purpose.
Add "omitempty" so that a `wsk action update` won't accidentally reset this value to the 0